core/src/main/scala/akka/persistence/cassandra/journal/CassandraJournal.scala

```diff
@@ -64,7 +63,7 @@ class CassandraJournal(cfg: Config, cfgPath: String) extends AsyncWriteJournal w
   private val log: LoggingAdapter = Logging(context.system, getClass)
 
   private implicit val ec: ExecutionContext = context.dispatcher
-  private implicit val materializer: ActorMaterializer = ActorMaterializer()(context.system)
+  private implicit val sys: ActorSystem = context.system
```

Comment:
Can just do import context.system as it is already implicit i think

Reply:
Done